packages/@okta/eslint-plugin-okta-ui/lib/rules/no-missing-api-keys.js

```diff
@@ -0,0 +1,55 @@
+const _ =  require('underscore');
+const loginEnBundle = require('../../../i18n/src/json/login.json');
+const { existsSync, readFileSync } = require('fs-extra');
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Detect missing i18nkeys from json mock files',
+      category: 'i18n Issues',
+      recommended: true,
+    },
+    messages: {
+      missingApiKeysLoginBundle: 'Missing i18n key from login.properties returned from the API. Add {{i18nKey}} to login.properties to prevent english leaks',
+      missingi18nKeyApiResponse: 'Missing i18n key in API mock {{file}}. Avoid using hard-coded strings returned and instead have the API return an i18n key and add that key to login.properties to prevent english leaks'
+    },
+  },
+  create(context) {
+    return {
+      'Program': function (node) {
+        const fileName = context.getFilename();
+          const jsonContent = JSON.parse(readFileSync(fileName, 'utf8'));
+          const messages = jsonContent.messages;
+          let loc = node.loc.start;
```

Comment:
When `loc` is not provided, it should [use the node value](https://eslint.org/docs/developer-guide/working-with-rules#contextreport). Given that, we should be able to remove this from the context.